exasol_script_languages_developer_sandbox/templates/vm_bucket_cloudformation.jinja.yaml

```diff
@@ -0,0 +1,32 @@
+Resources:
+  VMBucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
+      AccessControl: PublicRead
+    DeletionPolicy: Retain
+  BucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
+    Properties:
+      PolicyDocument:
+        Id: VMBucket_Policy
+        Version: 2012-10-17
+        Statement:
+          - Sid: PublicReadForGetBucketObjects
```

Comment:
Here we maybe need the cdn with rate limiting

Reply:
As discussed offline I changed the bucket to private for now. We can decide later if it should switched back to public, after we know the size of the VM.